File: exercises/small_problems_101/easy_2/when_will_i_retire.rb
require 'Date'

puts "What is your age?"
current_age = gets.chomp.to_i

puts "At what age would you like to retire?"
retire_age = gets.chomp.to_i

gap = retire_age - current_age
this_year = Date.today.year
retire_year = this_year + gap

puts "It's #{this_year}. You will retire in #{retire_year}"
puts "You have only #{gap} years of work to go!"

File: frontend/javascript/exercises/easy_3/when_will_i_retire.js
var age = Number(prompt('What is your age?'));
var ageToRetire = Number(prompt('At what age would you like to retire?'));
var years = ageToRetire - age;
var thisYear = new Date().getFullYear();
var yearToRetire = thisYear + years;

console.log("It's " + thisYear + '. You will retire in ' + yearToRetire + '.');
console.log('You have only ' + years + ' years of work to go!');